ipv-api/src/main/java/uk/gov/di/authentication/ipv/entity/IdentityProgressResponse.java

```diff
@@ -0,0 +1,57 @@
+package uk.gov.di.authentication.ipv.entity;
+
+import com.google.gson.annotations.Expose;
+import com.google.gson.annotations.SerializedName;
+import com.nimbusds.oauth2.sdk.id.State;
+import uk.gov.di.authentication.shared.validation.Required;
+
+import java.net.URI;
+
+public class IdentityProgressResponse {
+
+    @SerializedName("status")
+    @Expose
+    @Required
+    private IdentityProgressStatus status;
+
+    @SerializedName("client-name")
+    @Expose
+    @Required
+    private String clientName;
+
+    @SerializedName("redirect-uri")
+    @Expose
+    @Required
+    private URI redirectUri;
```

Comment:
Similarly, there isn't consistency between if we should return uri/state as strings or their given types - I'm happy to return strings if people think that's the correct option

Reply:
Happy with this. An approval test is a good way to check the serialisation if that's something you're concerned about

Reply:
Added one to check the above as well